guides/integrate-firebase.md

```diff
@@ -0,0 +1,272 @@
+
+Creating a user can be done either in the admin console or through an API. This guide will use the admin console. Navigate to your realm in the console and click **Identities**, then click **Add identity**.
+
+For more information visit [Workflows: User and Group Provisioning](/docs/v1/workflows/user-provisioning).
+
+<Arcade clip={Clip.CreateIdentity} />
+
+### Generate a passkey
+
+Once you have an identity you can generate a passkey. This step can also be done either in the admin console or through an API. This guide will use the admin console. Navigate back to **Identities** and select the identity you would like to bind to a passkey. Click **Add a passkey**, select your app and the click **Proceed & send email**. The user will receive an enrollment email which they can tap on to bind a passkey to their device.
+
+For more information visit [Workflows: Bind Passkey To User](/docs/v1/workflows/bind-passkey).
+
+<Arcade clip={Clip.CreatePasskey} />
+
+## Configure Application
+
+### Bind passkey to device
+
+Once the user taps on the enrollment email, they will be redirected to your application. Intercept the link from the enrollment email. The link that is redirected to your application will take on the following form. A `/bind` path will be appended to your Invoke URL (configured in your application above) as well as several other query parameters.
+
+```
+$invoke_url/bind?api_base_url=<api_base_url>&tenant_id=<tenant_id>&realm_id=<realm_id>&identity_id=<identity_id>&job_id=<job_id>&token=<token>
+```
+
+Once you receive the incoming URL, pass it into the SDK to complete the binding process. You can validate the incoming URL with `isBindPasskeyUrl`. Upon success, a private key will have been created in the device's hardware trust module and the corresponding public key will have been sent to the Beyond Identity Cloud. At this point the user has a passkey enrolled on this device.
+
+<Tabs groupId="sdks" queryString>
+<TabItem value="kotlin" label="Kotlin">
+
+```kotlin
+EmbeddedSdk.bindPasskey(url = bindingLink)
+.onEach { result ->
+    result.onSuccess { success ->
+        Timber.d("Bind Passkey success = $success")
+    }
+    result.onFailure { failure ->
+        Timber.e("Bind Passkey failure = $failure")
+    }
+}
+```
+
+</TabItem>
+<TabItem value="swift" label="Swift">
+
+```swift
+Embedded.shared.bindPasskey(url: bindingLink) { result in
+    switch result {
+    case let .success(bindResponse):
+        print(bindResponse)
+    case let .failure(error):
+        print(error.localizedDescription)
+    }
+}
+```
+
+</TabItem>
+</Tabs>
+
+### Authenticate
+
+The authenticate url that is redirected to your application will append a `/bi-authenticate` path to your Invoke URL:
+
+```
+$invoke_url/bi-authenticate?request=<request>
+```
+
+Once you receive the authenticate URL, pass it into the SDK to complete the authentication process. You can validate the incoming URL with `isAuthenticateUrl`.
+
+<Tabs groupId="sdks" queryString>
```

Comment:
For these code examples it might be helpful to show imports for the BI SDK and Firebase SDK

Reply:
Done.